### ISSUE TITLE
 java.lang.ClassCastException: java.lang.String cannot be cast to java.util.Map

### ISSUE THREAD

**Issue by Xgamefactory** [Baseflow/flutter-geolocator#280]
we are getting error after upgrading from 4.0.3
```
E/EventChannel#flutter.baseflow.com/geolocator/events(31800): 	at com.baseflow.geolocator.data.LocationOptions.parseArguments(LocationOptions.java:17)
E/EventChannel#flutter.baseflow.com/geolocator/events(31800): 	at com.baseflow.geolocator.tasks.TaskFactory.createStreamLocationUpdatesTask(TaskFactory.java:112)
E/EventChannel#flutter.baseflow.com/geolocator/events(31800): 	at com.baseflow.geol


```

using like in your example

```
import 'package:geolocator/geolocator.dart';

var geolocator = Geolocator();
var locationOptions = LocationOptions(accuracy: LocationAccuracy.high, distanceFilter: 10);

StreamSubscription<Position> positionStream = geolocator.getPositionStream(locationOptions).listen(
    (Position position) {
        print(position == null ? 'Unknown' : position.latitude.toString() + ', ' + position.longitude.toString());
    });
```

**Comment by yunyu** [comment]
I am experiencing the same issue, and am completely unable to use anything above 4.0.3

**Comment by mvanbeusekom** [comment]
Fixed in version 6.0.0
